### ISSUE TITLE
Error configuring zlib internal

### ISSUE THREAD

**Issue by meshula** [AcademySoftwareFoundation/openexr#933]
When using the zlib internal option, we set an alias in the cmake files, ZLIB:ZLIB. In order for that to work, the imports need to be declared as GLOBAL.

Currently we have

```cmake
add_library(zlib_static/shared STATIC/SHARED IMPORT)
```

to successfully build this option, we need
```cmake
add_library(zlib_static/shared STATIC/SHARED IMPORT GLOBAL)
```

Could someone with a working CLA patch this for me please?

**Comment by meshula** [comment]
@cary-ilm Could you patch this one in for us? This will resolve the various Windows zlib issues that have been posted.

**Comment by meshula** [comment]
https://github.com/AcademySoftwareFoundation/openexr/issues/871

**Comment by cary-ilm** [comment]
Check out #953.

**Comment by meshula** [comment]
Closed by #953
